tests/unittests/OnnxImporterTest.cpp

```diff
@@ -5437,3 +5437,127 @@ TEST_F(OnnxImporterTest, softmax13) {
                0.11920292, 0.880797, 0.880797, 0.11920292, 0.11920292, 0.880797,
                0.880797, 0.11920292, 0.11920292, 0.880797, 0.880797});
 }
+
+TEST_F(OnnxImporterTest, mixedPrecisionConfig) {
+  ExecutionEngine EE;
+  auto &mod = EE.getModule();
+  std::string netFilename(GLOW_DATA_PATH
+                          "tests/models/onnxModels/mixedPrecision.onnxtxt");
+  PlaceholderBindings bindings;
+  auto *F = mod.createFunction("main");
+  Placeholder *output;
+  Tensor data(ElemKind::FloatTy, {1, 3, 28, 28});
+
+  // Precision config info used in model loader
+  std::string precisionConfigFile(
+      GLOW_DATA_PATH "tests/models/onnxModels/mixedPrecision.yaml");
+  setModelLoaderPrecisionOpt(precisionConfigFile);
+
+  {
+    ONNXModelLoader onnxLD(netFilename, {"input"}, {&data.getType()}, *F);
+    output = EXIT_ON_ERR(onnxLD.getSingleOutput());
+    EXPECT_EQ(mod.getPlaceholders().size(), 2);
+    bindings.allocate(mod.getPlaceholders());
+  }
+  EE.compile(CompilationMode::Infer);
+  EE.run(bindings);
+  setConstantFoldLoaderOpsFlag(false);
+  auto result = bindings.get(output)->getHandle();
+  std::vector<dim_t> expectedDims = {1, 16, 26, 26};
+  EXPECT_TRUE(result.dims().vec() == expectedDims);
+
+  // This Model has three operators
+  // Here Conv and ReLu are specified in the yaml file to run in fp16.
+  //  Input: Placeholder(float)
+  //      |
+  //      V
+  //  Conv (set to float16)
+  //      |
+  //      V
+  //  BatchNorm(float)
+  //      |
+  //      V
+  //  ReLu( set to float16)
+  //      |
+  //      V
+  //  Output: Placeholder(float)
+  // Check in the Glow graph the corresponding nodes are in FP16.
+  for (auto &n : F->getNodes()) {
+    if (n.getName().str() == "Relu_2__1_max__1" ||
+        n.getName().str() == "Conv_0__2") {
+      EXPECT_TRUE(n.getElementType(0) == ElemKind::Float16Ty);
+    }
```

Comment:
nit: This will make the test fragile to how we unique node names and the format of the node. I would probably change this to something like:
```suggestion
    if (auto *RN = llvm::dyn_cast<ReluNode>(n)) {
      EXPECT_EQ(RN->getResult().getElementType(), ElemKind::Float16Ty)
    } else if (auto *CN = llvm::dyn_cast<ConvolutionNode>(n)) {
      EXPECT_EQ(CN->getResult().getElementType(), ElemKind::Float16Ty)
    }
```